nodejs/week2/mealsharing/src/backend/routes/meals.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+
+const meals = require(__dirname + '/../data/meals.json');
+
+// router.get('/meals', (request, response) => {
+//    response.send(meals);
+// });
+
+router.get('/:id', (req, res) => {
+   let id = req.params.id;
+   let meal = meals.find((items) => items.id === parseInt(id));
+   if (!meal) {
+      res.status(404).send(`The meal with the given ID: ${id} was not found`);
+   }
+
+   res.send(meal);
```

Comment:
How about putting it inside of `else`? If the meal is not found, you will call `send` twice on `res`.